app/inputs/ajax_filter_input.rb

```diff
@@ -1,11 +1,15 @@
-class AjaxFilterInput < Formtastic::Inputs::StringInput
+class AjaxFilterInput < Formtastic::Inputs::SelectInput
   include ActiveAdmin::Inputs::Filters::Base
 
   def to_html
     input_wrapping do
       [
         label_html,
-        builder.text_field(eq_input_name, input_html_options)
+        builder.select(eq_input_name,
+                        { get_selected_value => get_selected_id },
+                        { selected: get_selected_id },
+                        input_html_options
+                      )
```

Comment:
Indent ) the same as the start of the line where ( is.<br>Closing method call brace must be on the same line as the last argument when opening brace is on the same line as the first argument.